app/models/game.rb

```diff
@@ -0,0 +1,9 @@
+class Game < ActiveRecord::Base
+  has_many :memberships
+  has_many :users, through: :memberships
+
+  def <<(user)
```

Comment:
using such a generic method for such a specific use (adding a member) feels weird to me. what if Game evolves such that it's most array-like quality is no longer its memberships?